openapi/v1/integrationCurator.yaml

```
@@ -6699,6 +6700,49 @@ paths:
       summary: Retrieve file's metadata by study ID
       tags:
         - Files integration as Curator
+  /api/v1/as-curator/integration/link/cell/group/{sourceId}/to/sample/group/{targetId}:
+    post:
+      description: |-
+        Create links between cells and samples.
+
+        Cell objects of the same group can only be linked to objects of the same study.
```

```suggestion
        Cell objects from a source group can be linked multiple times, but only to target objects within the same study.
```
This sounds better to me

---

agreed on:
Cell objects of the same group can be linked to sample objects from multiple sample groups, but only within the same study.